spec/hardware/cpu.fmf

```diff
@@ -55,6 +55,9 @@ description: |
             - "!= flag4"
               ...
 
+          # Boolean, whether the cpu supports hyper-threading.
+          hyper: true| false
```

Comment:
My two cents: 'hyper' is one of the sooo overloaded buzzwords that I for one would rather see full form rather than "hyper". `hyperthreading` or `hyper-threading` would be way more readable to me, I wouldn't have to worry some CPU vendor comes up with yet another "hyper" technology :)

Reply:
One extra space left, it'd be nice to remove it.